crates/db_schema/src/aggregates/structs.rs

```diff
@@ -53,6 +54,7 @@ pub struct CommunityAggregates {
   pub users_active_month: i64,
   /// The number of users with any activity in the last year.
   pub users_active_half_year: i64,
+  pub hot_rank: i32,
```

Comment:
Why does community need a hot rank?

Reply:
For trending

Reply:
I honestly thought it just shows the most active communities. Not sure if the hot rank has any benefit there.